inst/builtin/bs5/shiny/_rules.scss

```diff
@@ -32,3 +34,30 @@ $web-font-path: "https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,
 .btn-default:not(.btn-primary, .btn-secondary, .btn-info, .btn-success, .btn-danger, .btn-warning, .btn-light, .btn-dark, .btn-link, [class*="btn-outline-"]) {
   @include button-outline-variant($secondary);
 }
+
+// This next section fixes inline checkbox and radio groups.
+// TODO: It would be nice to do this for all inline groups, but that would
+// require updating the input markup to use <fieldset> and <legend>.
+.shiny-input-container-inline {
+  .shiny-options-group {
+    display: flex;
+    display: -webkit-flex;
```

Comment:
These vendor prefixes were added by `tools/yarn_install.R` (sorry). Probably worth adding `inst/builtin` here? https://github.com/rstudio/bslib/blob/dcf4ae552e36d2ab0673ac0eb73d00bab7922fda/tools/yarn_install.R#L48

Reply:
Done in 23d6778a8dd2410744dc69f6a775f2aaadda3018 and I also added `inst/css-precompiled` to the list of ignored directories (since there's now a BS5 precompiled sass file there)

Reply:
oops, looks like we need `inst/css-precompiled` in that list so I put it back